01-node-tutorial/answers/03-modules.js

```diff
@@ -0,0 +1,6 @@
+const {name, age, address}  = require('./04-names');
+const greeting = require('./05-utils');
+const data = require('./06-alternative-flavor');
+
```

Comment:
Great job!
you can also require file `07-mind-grenade.js` here and when you run the file it will also run file 07 and print it's info.
```suggestion
require('./07-mind-grenade');
```